spec/API_specification/data_types.md

```diff
@@ -78,3 +78,29 @@ Implementations may provide other ways to specify data types (e.g.,
 
 A conforming implementation of the array API standard may provide and support additional data types beyond those described in this specification.
 ```
+
+(data-type-categories)=
+## Data Type Categories
+
+For the purposes of this spec, the following data type categories are defined.
```

Comment:
```suggestion
For the purposes of this specification, the following data type categories are defined.
```

To ensure consistent language across the specification, we've typically not relied on abbreviations.